environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - defaults
   - conda-forge  # Add other channels if needed
 dependencies:
-  - python=3.12  # Specify the Python version
+  - python=3.9  # Specify the Python version
```

Comment:
is there a reason we need to be on 3.9?? What's the error here?

Reply:
Line 268 in protein.py throws this error: SyntaxWarning: invalid escape sequence '\;'